stamp_generation/generator_input.py

```diff
@@ -0,0 +1,64 @@
+"""
+This script contains factory methods to get the input for generator
+This script can be removed later when the summarizer is ready.
+"""
+from classifier_and_summarizer.summarizer.summarizer_output import (StampPage,
+                                                                    StampPages)
```

Comment:
StampPage and StampPages should be in `data_models/stamp/`

Reply:
My bad. I missed it in PR review.

Reply:
I will do it in coming iterations. Thanks for the suggestion.